block_massaction.php

```diff
@@ -90,7 +94,7 @@ public function get_content(): stdClass {
         if ($this->page->user_is_editing()) {
 
             $applicableformatkey = 'course-view-' . $COURSE->format;
-            $iscoursecompatible = in_array($applicableformatkey, $this->applicable_formats())
+            $iscoursecompatible = in_array($applicableformatkey, array_keys($this->applicable_formats()))
```

Comment:
looks like a rebase to current `39_311_stable` would clean this up.

Reply:
thanks!